packages/css/card.css

```diff
@@ -13,178 +13,176 @@
-}
-
-.ds-card--brand2 {
-  --dsc-card-border-color: var(--ds-color-brand2-border-subtle);
-  --dsc-card-backround: var(--ds-color-brand2-surface-default);
-  --dsc-card-color: var(--ds-color-brand2-text-default);
-}
-
-.ds-card--brand2.ds-card--link {
-  --dsc-card-border-color: var(--ds-color-brand2-border-default);
-}
-
-.ds-card--brand2.ds-card--link:hover {
-  --dsc-card-backround: var(--ds-color-brand2-surface-hover);
-}
-
-.ds-card--brand2.ds-card--link:active {
-  --dsc-card-backround: var(--ds-color-brand2-surface-active);
-}
-
-.ds-card--brand3 {
-  --dsc-card-border-color: var(--ds-color-brand3-border-subtle);
-  --dsc-card-backround: var(--ds-color-brand3-surface-default);
-  --dsc-card-color: var(--ds-color-brand3-text-default);
-}
-
-.ds-card--brand3.ds-card--link {
-  --dsc-card-border-color: var(--ds-color-brand3-border-default);
-}
-
-.ds-card--brand3.ds-card--link:hover {
-  --dsc-card-backround: var(--ds-color-brand3-surface-hover);
-}
 
-.ds-card--brand3.ds-card--link:active {
-  --dsc-card-backround: var(--ds-color-brand3-surface-active);
+  &[data-link] {
+    --dsc-card-border-color: var(--ds-color-neutral-border-default);
+
+    text-decoration: none;
+
+    & h1,
+    & h2,
+    & h3,
+    & h4,
+    & h5,
+    & h6 {
+      color: var(--dsc-card-color);
+      text-decoration: underline;
+      text-decoration-thickness: max(1px, 0.0625rem, 0.1025em);
+      text-underline-offset: max(6px, 0.25rem, 0.22em);
+    }
+
+    &:hover {
+      --dsc-card-backround: var(--ds-color-neutral-background-subtle);
+    }
+
+    &:active {
+      --dsc-card-backround: var(--ds-color-neutral-surface-default);
+    }
+  }
+
+  &[data-color='subtle'] {
+    --dsc-card-border-color: var(--ds-color-neutral-border-subtle);
+    --dsc-card-backround: var(--ds-color-neutral-background-subtle);
+
+    &[data-link] {
+      --dsc-card-border-color: var(--ds-color-neutral-border-default);
+
+      &:hover {
+        --dsc-card-backround: var(--ds-color-neutral-surface-default);
+      }
+
+      &:active {
+        --dsc-card-backround: var(--ds-color-neutral-surface-hover);
+      }
+    }
+  }
+
+  &[data-color='brand1'] {
+    --dsc-card-border-color: var(--ds-color-brand1-border-subtle);
+    --dsc-card-backround: var(--ds-color-brand1-surface-default);
+    --dsc-card-color: var(--ds-color-brand1-text-default);
+
+    &[data-link] {
+      --dsc-card-border-color: var(--ds-color-brand1-border-default);
+
+      &:hover {
+        --dsc-card-backround: var(--ds-color-brand1-surface-hover);
+      }
+
+      &:active {
+        --dsc-card-backround: var(--ds-color-brand1-surface-active);
+      }
+    }
+  }
+
+  &[data-color='brand2'] {
+    --dsc-card-border-color: var(--ds-color-brand2-border-subtle);
+    --dsc-card-backround: var(--ds-color-brand2-surface-default);
+    --dsc-card-color: var(--ds-color-brand2-text-default);
+
+    &[data-link] {
+      --dsc-card-border-color: var(--ds-color-brand2-border-default);
+
+      &:hover {
```

Comment:
Skulle vi hatt `@media (hover: hover) and (pointer: fine) {` rundt alle `:hover`? 😊

Reply:
ja, good catch.